GameServer/Logic/RAF/RAFManager.cs

```diff
@@ -1,193 +1,131 @@
-﻿using InibinSharp;
-using InibinSharp.RAF;
-using LeagueSandbox.GameServer.Logic;
-using Microsoft.Win32;
-using System;
-using System.Collections.Generic;
+﻿using System;
 using System.IO;
-using System.Linq;
+using Newtonsoft.Json.Linq;
 
 namespace LeagueSandbox.GameServer.Core.Logic.RAF
 {
-    class RAFManager
+    static class RAFManager
```

Comment:
This class shouldn't be static. Ninject can handle objects in singleton scope, which is also what should be done here. Whenever a static class is used, parallelization becomes more difficult to achieve if wanted to. This is effectively removing the dependency injection aspect.